MAPIStoreFunctions.cpp

```diff
@@ -415,8 +416,11 @@ _Check_return_ HRESULT CreateStoreEntryID2(
 		sProps[2].ulPropTag = PR_FORCE_USE_ENTRYID_SERVER;
 		sProps[2].Value.b = true;
 
+		sProps[3].ulPropTag = PR_PROFILE_USER_SMTP_EMAIL_ADDRESS_W;
+		sProps[3].Value.lpszW = SmtpAddress.empty() == true ? nullptr : (LPWSTR)SmtpAddress.c_str();

```

Comment:
I removed the == true, but I received an error when I tried to remove the .empty().